src/Network/Network/FirewallPolicy/ManagedRules/ManagedRuleSet/ManagedRuleGroupOverride/ManagedRuleOverride/AzureApplicationGatewayFirewallPolicyManagedRuleOverride.cs

```diff
@@ -51,7 +58,8 @@ protected PSApplicationGatewayFirewallPolicyManagedRuleOverride NewObject()
             {
                 RuleId = this.RuleId,
                 State = string.IsNullOrEmpty(State) ? "Disabled" : this.State,
-                Action = this.Action
+                Action = this.Action,
+                Sensitivity = this.Sensitivity
```

Comment:
Since this sensitivity is only allowed for DDoS Ruleset, even though NRP has the validation, may be having the validation on PS side might be a better experience. I would recommend checking this with Powershell team once.

Reply:
Here is the sensitivity is null, it will still show as null sensitivity ie the sensitivity will show up as a an emoty entry for all other rulesets and this can lead to customer questions. Can we not display it if its null ?

Reply:
sensitivity won't show up if the customer don't put it in as a parameter